src/index.html

```diff
@@ -1,11 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
-</head>
-<body>
-    <h1>Hello webpack!</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta Name= "viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard</title>
+  </head>
+  <body>
+    <h1>Leaderboard</h1>
+    <section id="main">
+      <div class="item leaderboard-container">
+        <div class="container-title">
+          <h2>Recent scores</h2>
+          <div class="btn btn-refresh">
+            <button>Refresh</button>
+          </div>
+        </div>
+        <ul class="score-container">
+          <li>
+            <p class="name">Dotun</p>
+            <p class="score">100</p>
+          </li>
+          <li>
+            <p class="name">Fidelis</p>
+            <p class="score">20</p>
+          </li>
+          <li>
+            <p class="name">Barnarbas</p>
+            <p class="score">50</p>
+          </li>
+          <li>
+            <p class="name">Adeshina</p>
+            <p class="score">78</p>
+          </li>
+        </ul>
+      </div>
+      <div class="item form-container">
+        <form action="">
+          <h2>Add your score</h2>
+          <input type="text" id="name" name="name" placeholder="Your name" required/>
+          <input type="number" id="number" name="number" placeholder="Your score" required/>
+          <div class="btn btn-submit">
+            <button type="submit">Submit</button>
+          </div>
+        </form>
```

Comment:
- [ ] Kindly ensure your app layout matches the wireframe.

  Your layout | Wireframe layout
  :---:|:---:
  ![ddud](https://user-images.githubusercontent.com/54848363/208676992-372ba164-dc93-4c27-9d1b-7d1d4a59835d.jpg) | ![yydd](https://user-images.githubusercontent.com/54848363/208676997-34342f63-aa6c-4d3a-8cea-4f9361af66da.jpg)